data/data.csv

```diff
@@ -1,36 +1,41 @@
 COUNTRY,CURRENCY CODE,AVERAGE ANNUAL SALARY (WOMEN),AVERAGE ANNUAL SALARY (MEN)
-Algeria,DZD,4686,25655
-Angola,AOA,4708,8137
-Benin,CFA,2204,2329
-Botswana,BWP,16134,18604
-Burkina Faso,CFA,1484,2258
-Burundi,BIF,875,664
-Cameroon,XAF,3907,3481
-Cape Verde,CVE,4525,9156
-Chad,XAF,1476,2406
-Côte d'Ivoire,XOF,2542,5325
-Egypt,EGP,5102,17920
-Ethiopia,ETB,1428,2372
-Gambia,GMD,1013,2430
-Ghana,GHS,3837,5452
-Guinea,GNF,1813,2574
-Kenya,KES,2582,3998
-Lesotho,LSL,2333,3975
-Liberia,LRD,819,834
-Madagascar,MGA,1241,1871
-Malawi,MWK,954,1455
-Mali,XOF,1317,3105
-Mauritania,MRO,1994,5875
-Mauritius,MUR,14367,30350
-Morocco,MAD,3455,13075
-Mozambique,MZN,1150,1350
-Namibia,NAD,9812,11176
-Nigeria,NGN,4607,7081
-Rwanda,RWF,1649,2438
-Senegal,XOF,1828,3628
-South Africa,ZAR,9967,17160
-Swaziland,SZL,,
-Tanzania,TZS,2470,3432
-Tunisia,TND,5098,18889
-Uganda,UGX,1351,2383
-Zimbabwe,ZWL,1795,2393
\ No newline at end of file
+Algeria,DZA,3911.317698,23360.93433
+Angola,AGO,5369.422918,7345.432376
+Benin,BEN,2398.193453,2580.154093
+Botswana,BWA,12066.65863,15693.31307
+Burkina Faso,BFA,1325.544412,2198.505545
+Burundi,BDI,898.1065757,655.1730113
+Cameroon,CMR,2016.533099,4253.661935
+Cape Verde,CPV,5434.971681,6578.388613
+Chad,TCD,1351.433702,2152.754478
+Congo Dem Rep,COD,549.5414746,864.3387655
+Côte d'Ivoire,CIV,2571.122343,4653.344772
+Egypt,EGY,4833.657219,18413.27239
+Ethiopia,ETH,963.8840288,1659.39652
+Gambia,GMB,1214.545572,3419.409967
+Ghana,GHA,2598.492845,8343.627632
+Guinea,GIN,1809.062884,2639.403833
+Kenya,KEN,2544.393435,3616.085387
+Lesotho,LSO,2334.595263,3650.009061
+Liberia,LBR,1332.212742,1196.557722
+Madagascar,MDG,1249.270978,1505.486
+Malawi,MWI,821.7567033,1410.500586
+Mali,MLI,1318.658886,2637.88124
+Mauritania,MRT,1892.0477,5462.943148
+Mauritius,MUS,13809.30309,28519.17473
+Morocco,MAR,2966.617999,12626.43624
+Mozambique,MOZ,1018.351956,1352.370961
+Namibia,NAM,8751.834987,11119.93996
+Nigeria,NGA,4628.273515,6313.227139
+Papua New Guinea,PNG,2809.392381,3788.394817
+Rwanda,RWA,1439.362838,2354.026193
+Senegal,SEN,2143.936766,4628.137243
+Sierra Leone,SLE,1229.362976,1614.653405
+South Africa,ZAF,9239.829363,15131.35169
+Swaziland,SWZ,6838.753467,11758.10074
+Tanzania,TZA,2131.135139,2784.116074
+Togo,TGO,1103.400617,1835.753215
+Tunisia,TUN,4680.79841,17620.09567
+Uganda,UGA,1385.779816,2578.73306
+Zambia,ZMB,3755.849635,3687.303076
+Zimbabwe,ZWE,1852.106699,2515.70312
```

Comment:
New line

Reply:
Hi @kilemensi , I opted for a fork due to permission restrictions.

Reply:
Sorry for that @4bic ... I've added you to @CodeForAfrica/code-for-africa team.